spec/unit/views/components/status_tag_spec.rb

```diff
@@ -131,31 +131,12 @@ def status_tag(*args)
       end
     end
 
-    context "when status is 'Active' and type is :ok" do
-      subject { status_tag('Active', :ok) }
-
-      describe '#class_list' do
-        subject { super().class_list }
-        it      { is_expected.to include('status_tag') }
-      end
-
-      describe '#class_list' do
-        subject { super().class_list }
-        it      { is_expected.to include('active') }
-      end
-
-      describe '#class_list' do
-        subject { super().class_list }
-        it      { is_expected.to include('ok') }
-      end
-    end
-
     context "when status is 'Active' and class is 'ok'" do
       subject { status_tag('Active', class: 'ok') }
 
       describe '#class_list' do
         subject { super().class_list }
-        it      { is_expected.to include('status_tag') }
+        it      { is_expected.to include('status-tag') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.